stl/inc/xutility

```diff
@@ -5997,8 +6001,12 @@ _NODISCARD _CONSTEXPR_BIT_CAST bool _Is_inf(const _Ty _Xx) { // constexpr isinf(
 // FUNCTION TEMPLATE _Is_finite
 template <class _Ty, enable_if_t<is_floating_point_v<_Ty>, int> = 0>
 _NODISCARD _CONSTEXPR_BIT_CAST bool _Is_finite(const _Ty _Xx) { // constexpr isfinite()
+#if defined(__LDBL_DIG__) && __LDBL_DIG__ == 18
+    return _CSTD _LDtest(&_Xx) == _INFCODE;

```

Comment:
This will return `true` when `_Xx` is infinite, which I think is not what `_Is_finite` is intended to do. https://docs.microsoft.com/en-us/cpp/c-runtime-library/reference/finite-finitef?view=msvc-160 suggests this function returns `false` when the argument is an infinity or NaN.

Reply:
Thanks @CaseyCarter for the reviews. I checked wrong here. I have updated patch, so this will return true when _Xx is finite and false when the nfinity or NaN.